### DISCUSSION TITLE
Broken Module - Examples

### DISCUSSION THREAD

**Discussion by noor-codes** [keystonejs/keystone#7437] [Questions] [answered]
## Broken Module

Can anyone help me run the example repo, I seem to have a module related error.

Here is the repo: https://github.com/keystonejs/keystone/tree/main/examples/custom-field.
I have node 14 installed and running on Ubuntu.

![image](https://user-images.githubusercontent.com/28804764/161922064-0b21a48d-5e49-4cf8-8aae-eaf3c1fda3f3.png)

**Comment by fatihsolhan** [comment, marked as answer]
You need to add file extension as well on file path. './views.tsx'

**Reply by noor-codes** [reply]
Tried it, failed a couple of times. 
I made another clean repo and copied the code over and worked.

That's weird :(

**Reply by fatihsolhan** [reply]
If it says "Cannot find module", it is always a file path issue as you probably know. Sometimes you might need to restart the server after correcting the path. I'm glad it solved

**Reply by noor-codes** [reply]
Aha, The restarting might work next time. Thank You :)
